### PR TITLE
Add a release mechanism to push jar to the Central

### PR DESCRIPTION
**Pull request by interstellarPotato**

### CODE REVIEW

**Comment by interstellarPotato** [comment]
Not sure if PeaceMakr LLC will cut it under the 'developers' section. Perhaps.

**Comment by interstellarPotato** [comment]
To do: 
/ add a release mechanism for the Central via Gradle.

File: build.gradle [hunk 1]
```diff
@@ -47,4 +47,43 @@ task writePom {
...
+// Task for Sonatype Nexus OSS: build, sign, and upload
+uploadArchives {
+    repositories {
+        mavenDeployer {
+            // Sign POM
+            beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+            // Destination
+            repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+                authentication(userName: ossrhUsername, password: ossrhPassword)
```
Comment (notasecret) [inline build.gradle]: hmmm... placeholders?

File: gradle.properties [hunk 2]
```diff
@@ -1,2 +1,10 @@
...
\ No newline at end of file
+#target = android
+
+# See README.md 'How to release to maven central'
+signing.keyId={key_id}
+signing.password={key_passphrase}
+signing.secretKeyRingFile={secret_keys_ring_file}
+
+ossrhUsername={username}
+ossrhPassword={password}
```
Comment (notasecret) [inline gradle.properties]: ohhh, variables

File: build.gradle (hunk 1 above)
Comment (interstellarPotato) [reply build.gradle]: No, the package is pushed here, when the 'version' of the package does not contain '-SNAPSHOP' in the title. 
The package ends up in our private repo
Screen Shot 2020-02-07 at 2 33 10 PM

File: gradle.properties (hunk 2 above)
Comment (interstellarPotato) [reply gradle.properties]: Yeah, the prick is to use a local version gradle.properties. I have provided instructions in the readme.

**Pull request merged**